_docs/backenders/strategy-2021/roles/capacity.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Template
+parent: Roles
+grand_parent: Strategy 2021
+---
+
+# Capacity role definition
+
+**Current responsible**: -
+
+## Responsibility:
+    **Meetings and other FA:**
+- Facilitator on the capacity part of the BE SMO meeting.
+- Attend to capacity monthly meeting.
+- Participate in fortnightly BD meetings.
+- Participate in BD/FE retrospectives every 6 months.
+- Point of contact for other functional areas.
```

Comment:
@tiagojsag I know you added a note in the excel about this point. Would you like to have a more clear description for this responsability?

Reply:
Point of contact for other functional areas to request occasional support on small tasks (i.e.: DNS changes, restart servers, legacy project related work, etc).